toolbox/src/base/Signal.cpp

```diff
@@ -237,7 +234,7 @@ namespace wbt {
                 break;
         }
 
-        memcpy(contiguousData, address, dataSize * length);
+        memcpy(contiguousData, address, dataSize* length);
```

Comment:
revert this change. 
binary operators should have a space both before and after

Reply:
I quickly changed `<type> *var` to `<type>* var`, this is a mistake :sweat_smile: Anyway, I'll pass everything through a `clang-format` at the very end. Now it would make commits too dirty and difficult to squash / reorder